styles.css/main.css

```diff
@@ -105,40 +105,84 @@ nav ul li {
   display: block;
   position: relative;
 }
+.bio a {
+  display: block;
+  position: relative;
+}
 
-.paperclip {
+img.paperclip {
   background-image: none;
 
-  width: 50%;
-  position: relative;
-  left: -99px;
-  margin: auto;
+  border-radius: 0;
+  width: 20%;
+  position: absolute;
+  right: -7px;
+  top: -7px;
 }
 
-.favorite {
-  display: inline-flexbox;
+.favourites {
+  display: flex;
   color: rgb(203, 227, 255);
   width: 100%;
   flex-wrap: wrap;
   flex-direction: row;
+  justify-content: center;
+}
+.favourites > input {
+  display: none;  
+}
+.favourites > label {
+  order: -1;
+  margin: 1em;
+  opacity: 50%;
+}
+.favourites > input:checked + label {
+  opacity: 100%;
+}
+.favourites > div {
+  display: none;
+}
+.favourites > input:checked + label + div {
+  display: block;
+  width: 100%;
 }
 
 .gallery {
   display: block;
   display: grid;
-  grid-template-columns: 200px 200px 200px 200px;
-  justify-content: space-evenly;
-  padding: 10px 10px 10px 10px;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  gap: 10px;
```

Comment:
Use fractional units to create columns that adapt in width to the space available
Use the `gap` property to create grid gutters